### DISCUSSION TITLE
how to update snapshot name/description

### DISCUSSION THREAD

**Discussion by antiphy** [vmware/govmomi#3366] [Community Help] [answered]
I found create/remove/revert/list api, but no update snapshot name/description api. 
How can I update snapshot name/description via govmomi

**Comment by dougm** [comment, marked as answer]
The method is [RenameSnapshot](https://vdc-download.vmware.com/vmwb-repository/dcr-public/184bb3ba-6fa8-4574-a767-d0c96e2a38f4/ba9422ef-405c-47dd-8553-e11b619185b2/SDK/vsphere-ws/docs/ReferenceGuide/vim.vm.Snapshot.html#rename)
Currently the generated method is available, but looks like we don't have any govc command or helper methods. Untested example:
```go
	req := types.RenameSnapshot{
		This:        snapshotRef,
		Name:        "current-snapshot",
		Description: "The Current Snapshot",
	}

	_, err := methods.RenameSnapshot(ctx, vm.Client(), &req)
	if err != nil {
		return err
	}
```

**Reply by antiphy** [reply]
it works, thanks
